openapispec.json

```diff
@@ -0,0 +1 @@
ired":["offset","limit","total"]},"PersonNameParams":{"type":"object","properties":{"familienname":{"type":"string"},"vorname":{"type":"string"},"initialenfamilienname":{"type":"string"},"initialenvorname":{"type":"string"},"rufname":{"type":"string"},"titel":{"type":"string"},"anrede":{"type":"array","items":{"type":"string"}},"namenssuffix":{"type":"array","items":{"type":"string"}},"namenspraefix":{"type":"array","items":{"type":"string"}},"sortierindex":{"type":"string"}},"required":["familienname","vorname"]},"PersonBirthParams":{"type":"object","properties":{"datum":{"format":"date-time","type":"string"},"geburtsort":{"type":"string"}}},"PersonResponse":{"type":"object","properties":{"id":{"type":"string"},"referrer":{"type":"string"},"mandant":{"type":"string"},"name":{"$ref":"#/components/schemas/PersonNameParams"},"geburt":{"$ref":"#/components/schemas/PersonBirthParams"},"stammorganisation":{"type":"string"},"geschlecht":{"type":"string"},"lokalisierung":{"type":"string"},"vertrauensstufe":{"type":"string","enum":["KEIN","UNBE","TEIL","VOLL"]},"revision":{"type":"string"},"startpasswort":{"type":"string","description":"Initiales Benutzerpasswort, muss nach der ersten Anmeldung geändert werden"}},"required":["id","referrer","mandant","name","geburt","stammorganisation","geschlecht","lokalisierung","vertrauensstufe","revision","startpasswort"]},"CreatedPersonenkontextOrganisationDto":{"type":"object","properties":{}},"LoeschungResponse":{"type":"object","properties":{"zeitpunkt":{"format":"date-time","type":"string"}},"required":["zeitpunkt"]},"PersonenkontextResponse":{"type":"object","properties":{"id":{"type":"string"},"referrer":{"type":"string"},"mandant":{"type":"string"},"organisation":{"$ref":"#/components/schemas/CreatedPersonenkontextOrganisationDto"},"rolle":{"type":"string"},"personenstatus":{"type":"string"},"jahrgangsstufe":{"type":"string"},"sichtfreigabe":{"type":"string"},"loeschung":{"$ref":"#/components/schemas/LoeschungResponse"},"revision":{"type":"string"}},"required":["id","referrer","mandant","organisation","rolle","personenstatus","jahrgangsstufe","sichtfreigabe","loeschung","revision"]},"PersonendatensatzResponse":{"type":"object","properties":{"person":{"$ref":"#/components/schemas/PersonResponse"},"personenkontexte":{"type":"array","items":{"$ref":"#/components/schemas/PersonenkontextResponse"}}},"required":["person","personenkontexte"]},"CreatePersonBodyParams":{"type":"object","properties":{"username":{"type":"string"},"email":{"type":"string"},"referrer":{"type":"string"},"mandant":{"type":"string"},"stammorganisation":{"type":"string"},"name":{"$ref":"#/components/schemas/PersonNameParams"},"geburt":{"$ref":"#/components/schemas/PersonBirthParams"},"geschlecht":{"type":"string","enum":["m","w","d","x"]},"lokalisierung":{"type":"string","default":"de-DE"},"vertrauensstufe":{"type":"string","enum":["KEIN","UNBE","TEIL","VOLL"]},"auskunftssperre":{"type":"boolean"}},"required":["username","mandant","name"]}}}}
```

Comment:
Das File sollten wir nach `src/api-client` verschieben.

Reply:
Das File openapitools.json dann sinnvollerweise auch, oder?

Reply:
Wenn wir es verschieben können, gann gerne. Ich denke, das Config-File wird an genau dieser Stelle gesucht. Aber vielleicht kann man im Befehl in der package.json den Pfad zum Config-File angeben.

Reply:
Dazu gibt es ein offenes Ticket im Open Api CLI Github und anscheinend noch keine Lösung. Wir lassen es besser erst mal so.